### ISSUE TITLE
Error Updating the server

### ISSUE THREAD

**Issue by YvaGithub** [susanBuck/e15-spring22#21]
Hello Everyone,
After I install composer into the server, I run the alias "sites" to access the server for update.  By the way, my aliases were previously configured. I have gotten the following error:
![server_error](https://user-images.githubusercontent.com/42287520/155056072-981338f5-899a-4b28-af9a-dab8815d555f.JPG)

Any help will be greatly appreciated.
Yvaral

**Comment by susanBuck** [comment]
Hi @YvaGithub -

It looks like a connection related issue - can you try reconnecting? To do this, click the remote icon on the bottom left and choose "Close Remote Connection" from the menu that appears. Then you can reconnect.

FYI I did connect to your server and was able to run the `sites` alias without issue, so I don't think anything is broken/messed up.

Let me know if reconnecting doesn't do the trick, or if you continue to run into the same error.

**Comment by patrickgarsow-harvard** [comment]
Experienced something similar at one point and noted it happened after I did an update the VSCode wanted me to do. I rebooted my system as it was due anyways and it seemed to solve the problem. Seeing that the instructor can get into the server I think it definitely points to client side but you could also try restarting your droplet on digital ocean. Good luck!

**Comment by archerdave** [comment]
Ah, I saw something similar myself.  In my case, I am working from a Mac, and I had to re-run the fix that was mentioned in the first week of notes.   One sec ... It was these notes on [installing 'code' command in Path](https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line).

**Comment by susanBuck** [comment]
Hi @YvaGithub - Any updates on this? Is it working now? Let us know if we can provide any additional help.

**Comment by YvaGithub** [comment]
Hi,
Thank you for following up with me. The very next day after the issue I
tried to close it because it worked. However, I kept getting a message from
GitHub that I could not post a comment at that time. And I have been
noticing the same thing happen several times over time. When GitHub would
not work, I tried to email you directly, but the work computer would not
let me either. Sorry, I should have tried again from home.
Thank you,
Yvaral
